system-test/fixtures/sample/src/index.js

```diff
@@ -20,7 +20,7 @@
 const webrisk = require('@google-cloud/web-risk');
 
 function main() {
-  const webRiskServiceClient = new webrisk.WebRiskServiceClient();
+  const webRiskServiceV1Beta1Client = new webrisk.WebRiskServiceV1Beta1Client();
```

Comment:
@bcoe I don't understand why this got changed to the V1Beta1 version of the client.